js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -512,19 +512,21 @@ const RightPanelDetail = ({
 
 const MiddlePanel = ({
   assetKey,
-  selectedEvaluationId,
+  selectedEvaluation,
   maxMaterializationsPerMinute,
 }: {
   assetKey: Omit<AssetKey, '__typename'>;
-  selectedEvaluationId?: number;
+  selectedEvaluation?: any;
   maxMaterializationsPerMinute: number;
 }) => {
   const {data, loading, error} = useQuery<GetEvaluationsQuery, GetEvaluationsQueryVariables>(
     GET_EVALUATIONS_QUERY,
     {
       variables: {
         assetKey,
-        cursor: selectedEvaluationId ? (selectedEvaluationId + 1).toString() : undefined,
+        cursor: selectedEvaluation?.evaluationId
```

Comment:
it's a bit janky that we do this query twice, but I left it in because it handles the db migration error. Probably an easy refactor to avoid it for a non noob

Reply:
Its actually intentional. Because when you paginate forward/backwards, its possible that the evaluation you're look at in the middle panel is not in the list on the left nav.

Reply:
which I think is what this does, but I'll test